test/functional/tests/memory/test_memory_consumption_full_cache.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheLineSize, CacheMode
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.os_utils import (get_mem_available_from_meminfo,
+                                 get_vmalloc_cas_cache_used_from_vmallocinfo,
+                                 is_memory_within_limit,
+                                 validate_memory_consumption,
+                                 get_metadata_max_size)
+from test_utils.os_utils import sync
+from test_utils.size import Size, Unit
+
+cores_count = 16
+cache_mode_wb = CacheMode.WB
```

Comment:
If cache mode is significant, please mention it in title and/or description of the test.